### PR TITLE
Update caption

### PR DESCRIPTION
**Pull request by gunpun1**

### CODE REVIEW

**Review by matthew-fogel — APPROVED** [review]
Approved

File: _slides/08-gunpun1.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hello to you all!
```
Comment (CalinL) [inline _slides/08-gunpun1.md]: All great, perfect 👍

Comment (CalinL) [inline _slides/08-gunpun1.md]:
```suggestion
Hello to you all! I think looks better with new message.
```

**Review by CalinL — APPROVED** [review]
LGTM 🥇

**Pull request merged**